data/hai/unfettered 1 alphas.txt

```diff
@@ -0,0 +1,1533 @@
+			`	You seem to have stressed her a bit. "Are you always in a rush?" she asks. "Just look for traces of these particles; it should lead you to them. Based on chemical composition, these particles seem to be coming from the jump drive - it must be malfunctioning, but I have no idea in what way it is. This Auxiliary is no joke - it has room for over one hundred crew members, so be sure to have what it takes to capture it. You could always disable it and then come back later to board them; but in that case, make sure to be quick, or else the Alphas could repair it." You remember it also had fighters accompanying it.`
+				goto agree
+			label capture
+			`	She takes a while to answer, "We never focused too much on capturing ships, preferring to see the enemy eye to eye and only stun them, so they may live to fight another day but I'm sure you humans may have... alternative approaches. Their base is probably isolated, so you could always disable them and come back later with a different ship, or just outfit the one you have differently, or even take a Centipede with your fighting ship."`
+			choice
+				`	"What about the damage on their ship? How is that relevant to my mission?"`
+					goto information
+				`	"Can you tell me more about what kind of damage it was, on their ship?"`
+			label damage
+			`	"The damage was done by some kind of beam weapon, and a very strong one at that. It looks like it only got partially deflected by the shields, if at all. I wonder what kind of weapon can do that." She looks intrigued, and pauses to think about it, then continues, "That would explain why it's forbidden for us to venture east with jump drives. We lack any hull repair capabilities, and to be honest with you, our hull is not much more resilient than the one of some human ships, like that Auxiliary."`
+			choice
+				`	"They seemed to be using the same weapon against me."`
+					goto logs
+				`	"What can we use that information for?"`
+					goto information
+			label logs
+			`	"Yes, I noticed on the scanner logs. I would advise not relying too heavily on your shields, but they don't seem to have that many of those weapons, so you should be fine. I do not believe they could have designed such weapons by themselves. They probably just stole them from nearby aliens."`
+			choice
+				`	"What about the damage on their ship? How is that relevant to my mission?"`
+					goto information
+				`	"You should have asked me to do an outfit scan, earlier."`
+					to display
+						not "unfettered: scanned auxiliary's outfits"
+				`	"I did well to scan its outfits anyway then, so we have some information to work with."`
+					to display
+						has "unfettered: scanned auxiliary's outfits"
+					goto scanned
+			`	"Maybe, but it could have provoked them further. They apparently like their secrets. I only wanted to know about the structure of the ship, but I'd agree that knowing more about those weapons would be useful. Now, back to the damage on their ship.`
```

Comment:
```suggestion
			`	"Maybe, but it could have provoked them further. They apparently like their secrets. I only wanted to know about the structure of the ship, but I'd agree that knowing more about those weapons would be useful. Now, back to the damage on their ship."`
```